api/src/controllers/dashboard-controller.js

```diff
@@ -0,0 +1,21 @@
+import dashboardService from '../services/dashboard-service';
+
+export default class dashboardController {
+
+    static register(app) {
+
+        app.get('/dashboard/:userId', function (req, res) {
+            const  userId  = req.params['userId'];
+console.log('a')
+console.log(userId, req.params);
+            dashboardService.getUserPosts(userId, function (results) {
```

Comment:
can we use arrow functions instead of anonymous function declarations? I think they're prettier :)